synfig-studio/src/gui/app.cpp

```diff
@@ -2313,6 +2320,7 @@ App::quit()
 	process_all_events();
 
 	Gtk::Main::quit();
```

Comment:
As Gtk::Main is deprecated, this shouldn't be called.

And `quit()` method is/was to make the application quit, and not to be run "on shutdown". Do you understand what I mean? :)

Reply:
Understood :) I was just experimenting it ! Actually the main error I am having while overriding on_shutdown is ``` ‘void studio::App::on_shutdown()’ marked ‘override’, but does not override ``` So, I was just trying to deal with this error ! 

Moreover I understood the part you are explaining abt on_shutdown ! thanks :)